tests/data/valid/error/payload.json

```diff
@@ -245,5 +245,8 @@
                 "message": "Cannot read property 'baz' of undefined"
             }
         }
-    ]
+    ],
+    "transaction": {
```

Comment:
I believe this should be inside the elements in `errors`. Agents can buffer up errors from different transactions

Reply:
ugh, my mistake, that makes much more sense.